backend/dal/Migrations/20210908232309_v0.2.0.9.1.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
```

Comment:
The only issue I see with this migration is that when I do one for the leases I think mine would be v0.02.09.0. But I'm not certain.  All I know is we probably want the one that goes to each environment to match the release version.

Reply:
Just realized I think we're in `v0.2.10.0`, so that should make my migration work fine.